src/main/java/bio/terra/stairway/FlightMap.java

```diff
@@ -110,8 +109,23 @@ public void fromJson(String json) {
     }
   }
 
+  // Truncate working map to only print first 500 characters
   @Override
   public String toString() {
-    return new ToStringBuilder(this, ToStringStyle.JSON_STYLE).append("map", map).toString();
+    int truncateLength = 500;
+    StringBuilder sb = new StringBuilder("{");
+    sb.append(
+        map.entrySet().stream()
+            .map(
+                set -> {
```

Comment:
This is such a minor thing, but I would call this `entry` not `set` since it's an entry in the Map (and its type is `Map.Entry`).